tests/kuttl/tests/dataplane-deploy-tls-test/02-dataplane-deploy-services-override.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: custom-service1
```

Comment:
Can we rename these services appropriately as `custom-service1` and `custom-service-2` are for different scenarios? What about scenario of default  services in NodeSet?

Reply:
@rabi  sorry.  I'm not quite sure I understand what you are asking for here.

The TLS cert functionality needs services that request a TLS certificate by having a tlsCert attribute.  

custom-service1 and custom-service2 are examples of these kinds of services - just with slightly different contents requested in the tlsCert.  I'm not sure why we want to name them differently.  If we decide later to add another service to test some different cert content - like a non-default issuer , for instance, how should we name those?

Unless maybe you're asking for the services to be named more explicitly - like:
service-tls-dnsnames-ips-ctlplane-rootca-internal-issuer
service-tls-dnsnames-all-networks-default-issuer

These can get pretty long, but they would describe more explicitly whats being tested.  Though I suppose you can always look at the test to see whats failing.

As for default services, the reason I use custom services is because I don't think nodes are actually created when we run kuttl tests.  We need nodes to be defined in the nodeset with actual ips and dnsnames to generate the certs.  We then check that the certs are generated and mounted properly by the install-certs-override service.  But when the services run, we want to be sure that they don't actually reach out to the non-existent nodes (or the test will fail).  So they run a play that sleeps on localhost.

I think what you're asking for here is for me to create a new default-service with a tls-cert attribute  (that sleeps on localhost) , add that to my definition of the nodeset as a default service, and then do a deployment without service-overrides?

Reply:
ok -- I think I addressed your comments .. lmk :)